docs/frontend-shared.md

```diff
@@ -1,67 +1,39 @@
 # Frontend Shared Package
 
-
 The `frontend-shared` package is a library of shared utilities, components and styles common to multiple Hypothesis front-end applications.
 
-The `frontend-shared` package source is located in `./frontend-shared` in this repository.
-
-Note: The long-term goal is to move this package into its own repository. There are some short-term advantages to the current setup: it piggy-backs on existing tooling and gives developers the benefit of working with just one repository when making changes (which primarily benefit the `client` in the first place).
-
-## How it works
-
-Frontend-shared works by creating a symlink via `$ yarn link` between the working dir
-`./frontend-shared/` and the installed dir `node_modules/@hypothesis/frontend-shared/`.
-
 ## Client setup
 
-To set up `frontend-shared` in the client, run
+To set up ``frontend-shared`` in the client, first build the package and then publish to a your local yalc repository. You may need to install yalc first. ``yarn global add yalc ``
```

Comment:
I suggest extracting the "You may need to install yalc first" sentence into a separate step above this one, so that the overall sequence of steps is:

1. Install the `yalc` tool globally
2. Publish the package to the local yalc repository
3. Use the copy of the package from the yalc repo in the project(s) where you want to test your changes

The section about yalc would also benefit from a very brief explanation of what yalc is. This can just be something like `[yalc](link) is the tool that we use for locally making changes to a shared package and testing the changes in another project`.